### PR TITLE
add sidebar (Pop up) and geoSearch feature

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

File: src/components/manage/Widgets/GeolocationWidget.jsx [hunk 1]
```diff
@@ -2,11 +2,13 @@ import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Grid } from 'semantic-ui-react';
 import { defineMessages, injectIntl } from 'react-intl';
-import { FormFieldWrapper } from '@plone/volto/components';
+import { FormFieldWrapper, CheckboxWidget } from '@plone/volto/components';
 import { unionBy } from 'lodash';
+import SidebarPopup from '@eeacms/volto-block-style/SidebarPopup/SidebarPopup';
```
Comment (avoinea) [inline src/components/manage/Widgets/GeolocationWidget.jsx:7]: Please add this dependency to README and DEVELOP

File: src/components/manage/Widgets/SearchGeoName.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import InlineForm from 'volto-slate/futurevolto/InlineForm';
```
Comment (avoinea) [inline src/components/manage/Widgets/SearchGeoName.jsx]: Please add `volto-slate` dependency to README and DEVELOP

Comment (nileshgulia1) [reply src/components/manage/Widgets/SearchGeoName.jsx]: Sure.

File: src/components/manage/Widgets/GeolocationWidget.jsx (hunk 1 above)
Comment (nileshgulia1) [reply src/components/manage/Widgets/GeolocationWidget.jsx:7]: Sure.

**Pull request merged**